e2e/test_util.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from src.files import NetworkFile, File
+
+TEST_FILE = File(name="test-file.csv", path='./out/test')
+TEST_NETWORK_FILE = NetworkFile(name="test-network-file.csv", path='./out/test')
+
+
+def drop_files(func):
+    def wrapped_func(self):
+        func(self)
+        try:
+            os.remove(TEST_NETWORK_FILE.filename('mainnet').filename())
+            os.remove(TEST_FILE.filename())
```

Comment:
If the first file doesn't exist the second file won't get deleted. I assume this normally shouldn't happen but one could prevent that and have more extendable code by iterating over a list of file names and doing
```python
try:
    os.remove(filename)
except FileNotFoundError:
    pass
```
for every one.